scripts/install

```diff
@@ -118,7 +118,7 @@ configure_netdata
 # INTEGRATE SERVICE IN YUNOHOST
 #=================================================
 
-yunohost service add $app --description "Real-time performance and health monitoring for systems and applications" --log "$final_path/var/log/netdata/error.log" "$final_path/var/log/netdata/access.log" "$final_path/var/log/netdata/debug.log"
+yunohost service add $app --description="Real-time performance and health monitoring for systems and applications" --log="$final_path/var/log/netdata/error.log" "$final_path/var/log/netdata/access.log" "$final_path/var/log/netdata/debug.log"
```

Comment:
Hmm after testing it doesn't appear to work and doesn't correspond to help:
```
usage: yunohost service add name
                            [-h] [-d DESCRIPTION] [-l LOG [LOG ...]]
                            [-t {file,systemd} [{file,systemd} ...]]
                            [--test_status TEST_STATUS]
                            [--test_conf TEST_CONF]
                            [--needs_exposed_ports PORT [PORT ...]] [-n]
                            [-s STATUS]

Add a service

positional arguments:
  name                  Service name to add

optional arguments:
  -h, --help            show this help message and exit
  -d DESCRIPTION, --description DESCRIPTION
                        Description of the service
  -l LOG [LOG ...], --log LOG [LOG ...]
                        Absolute path to log file to display
  -t {file,systemd} [{file,systemd} ...], --log_type {file,systemd} [{file,systemd} ...]
                        Type of the log (file or systemd)
  --test_status TEST_STATUS
                        Specify a custom bash command to check the status of
                        the service. Note that it only makes sense to specify
                        this if the corresponding systemd service does not
                        return the proper information already.
  --test_conf TEST_CONF
                        Specify a custom bash command to check if the
                        configuration of the service is valid or broken,
                        similar to nginx -t.
  --needs_exposed_ports PORT [PORT ...]
                        A list of ports that needs to be publicly exposed for
                        the service to work as intended.
  -n, --need_lock       Use this option to prevent deadlocks if the service
                        does invoke yunohost commands.
  -s STATUS, --status STATUS
                        Deprecated, old option. Does nothing anymore. Possibly
                        check the --test_status option.
```